phpunit/class-wp-rest-global-styles-controller-gutenberg-test.php

```diff
@@ -79,14 +79,18 @@ public static function wpTearDownAfterClass() {
 	 * @covers WP_REST_Global_Styles_Controller_Gutenberg::register_routes
 	 */
 	public function test_register_routes() {
+		// Register routes so that they overwrite identical Core routes.
+		$global_styles_controller = new WP_REST_Global_Styles_Controller_Gutenberg();
+		$global_styles_controller->register_routes();
```

Comment:
This needs to occur to ensure we're only testing the Gutenberg-registered endpoints...